kbase-extension/static/kbase/js/widgets/appWidgets2/input/select2ObjectInput.js

```diff
@@ -175,7 +175,8 @@ define([
                     processedValue = '',
                     validationOptions = {
                         required: spec.data.constraints.required,
-                        authToken: runtime.authToken(),
+                        // Need to validate that runtime.authToken exists. Throwing error in console while testing select2spec because it doesn't exist. Evaluates to null
+                        //authToken: runtime.authToken(),
```

Comment:
When running the FE tests with a slow internet connection, runtime.authToken() fails.

I _think_ because this call fails, the promise in `validate()` is never returned. The promise is created via Bluebird.